wordpress-mobile/WordPress-Android
# Issue/skip blogging prompt

This PR allows skipping of blogging prompt.

Skipping logic is based on a specific day stored in app preferences.

To test:
- Enable BP feature flag.
- From dashboard BP card, tap on the menu button and press "Skip this prompt"
- Confirm that the card disappears and snackbar with undo button is visible.
- Press "Undo" and confirm that card reverts back.
- Move the date to one day ahead and refresh the dashboard.
- Confirm that the card is there.

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
